eosio_build.sh

```diff
@@ -120,7 +120,7 @@
 		export SOFTFLOAT_ROOT=/usr/local/berkeley-softfloat-3
 		CXX_COMPILER=clang++
 		C_COMPILER=clang
-		BUILD_MONGO_DB_PLUGIN=true
+		BUILD_MONGO_DB_PLUGIN=false
```

Comment:
I believe we need this enabled on Jenkins darwin build so we test the mongo plugin on darwin.

Reply:
Yes, that will be in my next PR which Im working on now.